main/EventSystem.cpp

```diff
@@ -3301,54 +3336,59 @@ void CEventSystem::EvaluateLua(const std::string &reason, const std::string &fil
 		secstatusw = "Disarmed";
 	}
 
-	std::stringstream lua_DirT;
-
-#ifdef WIN32
-	lua_DirT << szUserDataFolder << "scripts\\dzVents\\";
-#else
-	lua_DirT << szUserDataFolder << "scripts/dzVents/";
-#endif
-
 	lua_createtable(lua_state, 7, 0);
```

Comment:
As the table size would now conditional on dzVents being enabled, this should probably be lua_createtable(lua_state, 0, 0);